### ISSUE TITLE
EditorContent not rendering in Quasar 2 (Vue 3)

### ISSUE THREAD

**Issue by mfoitzik** [ueberdosis/tiptap#1606]
**Description**
Using the basic setup instructions (https://www.tiptap.dev/installation/vue3) the EditorContent component does not render. However it is created as I can reference it and access its methods and for example use editor.getHTML().

If I bind an element instead of using EditorContent, it renders just fine.

**Steps to reproduce the bug**
See repo reference below

**Repo**
I created a project that is located at https://github.com/mfoitzik/quasar-tiptap-test

**Expected behavior**
clone repo
yarn
quasar dev

The main page has two links. One link is to the EditorContent test the other is to the BoundElement test. 

**Screenshot, video, or GIF**

**Environment?**
 - operating system: Windows 10 Pro
 - browser: Firefox/Chrome
 - mobile/desktop: desktop
 - tiptap version: @tiptap/starter-kit: "^2.0.0-beta.82", @tiptap/vue-3: "^2.0.0-beta.45",

**Comment by mfoitzik** [comment]
I ended up getting this to work based on @philippkuehn answer in https://github.com/ueberdosis/tiptap/issues/1110

However, I needed to add a bit more specific resolution:
```
"resolutions": {
    "vue": "^3.1.4"
  }
```
then remove the yarn.lock file and then run yarn install

The current version of quasar seems to specifically call for vue 3.1.2

I worked backwards from the working vue-3 sample instructions (which works fine) at https://www.tiptap.dev/installation/vue3/ which currently (as of today - 7/16/2021) loads vue 3.1.4. If I downgrade that setup to vue 3.1.2 it will fail in the same way (EditorContent does not render).

I updated my test repo at https://github.com/mfoitzik/quasar-tiptap-test just in case it may help someone else get started with Quasar 2.
